### ISSUE TITLE
Fix scipy dependency

### ISSUE THREAD

**Issue by YAY-C** [lcmd-epfl/Q-stack#41]
Deprecation Warning from latest update
```
DeprecationWarning: The 'sym_pos' keyword is deprecated and should be replaced by using 'assume_a = "pos"'. 'sym_pos' will be removed in SciPy 1.11.0.
```

**Comment by briling** [comment]
@YAY-C could you please check if it's still an issue with the new packaging

**Comment by YAY-C** [comment]
yep, the problem has been solved (by @briling ahah)!

solved in 96748e8
